scripts/travis/install-requirements.sh

```diff
@@ -1,8 +1,13 @@
 #!/usr/bin/env bash
 
-if [ "${TASK}" = 'compilepy3 ci-py3-unit' ] || [ "${TASK}" = 'ci-py3-integration' ]; then
+if [[ "${TASK}" = *'ci-py3'* ]]; then
```

Comment:
Please revert to more explicit match as there is a chance to include irrelevant tasks in future.

Reply:
And I want to make sure that it catches all py3 tasks here as they all run in tox. I don't want to have to update this line repeatedly if we move the jobs around (as @blag suggested we need to do in another PR with the lint checks moving to python3).

Reply:
Changing the targeted tasks here makes it very clear that this is expected to run for all py3 tasks and that it is because they are py3 that we're targeting them separately, not just because they're a random selection of tasks.

Reply:
If you're going to keep this in, it needs to be commented in somewhere, like `jobs.include`. And you can even include a link to this file in those comments so people can check for themselves.

We should definitely leave breadcrumbs for the poor soul who has to update this in the future.

Reply:
Yes. It was not the nicest thing to go grepping through to find out where the task names had been hard coded so I could change one of them. I'll add a comment to .travis.yml on where the task names are used.

Reply:
commented in relevant places